chore: Add basic info about how Docker images are built (#81)
getsentry/develop: src/docs/continuous-integration.mdx

- Deploy the fix, and verify that the issue has indeed been fixed.
- Inform #team-engineering that it is safe to deploy getsentry again.

## Docker images

> We use Google Cloud Build to build our Docker containers. Every push to master additionally gets build and uploaded to Docker Hub:

Reviewer: ```suggestion
We use Google Cloud Build to build our Docker containers. Every single commit (including from PRs) gets built, tested and becomes available at `us.gcr.io/sentryio/sentry:<sha>` with no retention guarantees. Every push to `master` gets built, tested and [pushed to Docker Hub](https://hub.docker.com/r/getsentry/sentry/tags) permanently. You can sent `SENTRY_IMAGE` to the following, when working with [getsentry/onpremise](https://github.com/getsentry/onpremise):
```

> * `getsentry/sentry:commit_sha` as image to pull from a commit in `getsentry/sentry` master
> * `us.gcr.io/sentryio/sentry:commit_sha` as image to pull from a commit in any other branch

Reviewer: ```suggestion
* `getsentry/sentry:<sha>`, `getsentry/sentry:<short_sha>`, or `getsentry/sentry:latest` to pull from a commit in [getsentry/sentry](https://github.com/getsentry/sentry) `master`
* `us.gcr.io/sentryio/sentry:<sha>` to pull from a commit in any other branch or PR
```

> This is particuarly useful to know when working with [onpremise](https://github.com/getsentry/onpremise), where those values can be passed in as `SENTRY_IMAGE`.
> 

Reviewer: ```suggestion
```

**Solution:**

> Purge the travis cache. How? See above.

Reviewer: ```suggestion
Purge the Travis cache. How? See above.
```
